lib/v2/screens/dashboard/token_cards_widget.dart

```diff
@@ -12,50 +12,16 @@ import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/token_balances_
 
 import 'components/currency_info_card_widget.dart';
 
-// TODO(n13): Make this stateless
-class WalletHeader extends StatefulWidget {
-  const WalletHeader({
-    Key? key,
-  }) : super(key: key);
-
-  @override
-  State<StatefulWidget> createState() {
-    return WalletHeaderState();
-  }
-}
-class WalletHeaderState extends State<WalletHeader> {
+class TokenCardsWidget extends StatelessWidget {
   final CarouselController _controller = CarouselController();
-  int _selectedIndex = 0;
```

Comment:
❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️

Reply:
remove all the states from the view!!!

Reply:
Not sure how is possible to remove a controller from here.

Since this screen use this controller it must be stateful widget.